core/src/model/IPlayerCharacter.java

```diff
@@ -25,6 +27,19 @@ public interface IPlayerCharacter {
     void gainExperience(int experience);
     void reduceExperience();
     void increaseCurrentHealth(float amount);
+    void increaseLevel();
+    void decreaseLevel();
```

Comment:
Not used, unsure when it would be used in this videogame, only usable if player tpd to a certain place everytime you died. Doesnt remove perks either so it will have no effect currently